src/components/settings/Theme.tsx

```diff
@@ -27,7 +27,19 @@ const ThemeSettings = () => {
                 <label>Background Image</label>
                 <input
                     type="file"
-                    onChange={(e) => setSettingValue('themeBackgroundImagePath', e.target.value)} value={settings.themeBackgroundImagePath} />
+                    accept='image/*'
+                    onChange={(e) => {
+                        const files = e.target.files ?? [];
+                        const fileName = files[0];
+                        const fileData = new FileReader();
+                        let b64Image;
+                        fileData.onload = (e) => {
+                            const result = e.target?.result as ArrayBuffer;
+                            b64Image = btoa(new Uint8Array(result).reduce((data, byte) => data + String.fromCharCode(byte), ''));
+                            setSettingValue('themeBackgroundImageBase64', b64Image);
+                        };
+                        fileData.readAsArrayBuffer(fileName);
```

Comment:
might be better to create separate function so that this can be just `loadImageAsBase64(filePath).then((base64Image) => setSettingValue('themeBackgroundImageBase64', base64Image))